### DISCUSSION TITLE
Fancybox5 strips <script> tags from ajax windows

### DISCUSSION THREAD

**Discussion by IlIMiniReiDIlI** [fancyapps/ui#553] [Q&A]
Hello, Fancybox5 strips <script> tags from ajax windows

Fancybox4 works correctly with this.

How to make Fancybox5 not cut scripts?

**Comment by fancyapps** [comment]
Hi,

Running all scripts is a security risk. Please use events - https://fancyapps.com/fancybox/api/events/ - to execute your code as needed.

**Comment by IlIMiniReiDIlI** [comment]
This is bad, because all the connected scripts pass through our traffic using the backend and all that is possible is to display them.
